scripts/coverage_diff.py

```diff
@@ -90,15 +90,13 @@ def parse_coverage_status(coverage):
     coverage: {filename: {parent_cov: float, child_cov: float, delta_cov: float, new_file: bool}}
     output: {"exit_code":int, "message":str} 
     """
+    latest_non_terminate_status = {"exit_code":0, "message":"Coverage non decreasing.", "log_level":"notice", "terminate":False}
```

Comment:
This includes a `log_level`, but it is not used anywhere. That hints you something is not quite right with the organization of the functions, see next comment.